index.markdown

```diff
@@ -15,7 +15,7 @@ target="base-repo/index.html"
 background="assets/images/disks.jpg"
 title="base-repo"
 subtitle="A place to be for your data"
-content="The base-repo is a generic, general purpose research data repository service offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
+content="The base-repo is a generic, general purpose research data repository service. It is offering clear, machine-actionable RESTful interfaces for storing, retrieving and managing research data."
```

Comment:
It offers a clear...?

Reply:
I tried to keep the plural as I tried not to change the meaning. Should I change it?
I could make "It offers clear, machine, actionable RESTful interfaces for ...", keeping the plural and remove the -ing-form. I like the simplification. But before that, should it be described as a single interface or multiple interfaces?

Reply:
If multiple interfaces are offered, I would go with your revised phrasing.

Reply:
Stay with the plural, that's fine. Alternatively, one could remove RESTful as there is also the possibility to enable an OAI-PMH interface, which is not RESTful, but which justifies the plural even if all endpoints belonging to base-repo are interpreted as one interface. ;-)